js/ui/handler/scroll_zoom.js

```diff
@@ -97,6 +97,7 @@ ScrollZoom.prototype = {
     },
 
     _zoom: function (delta) {
+        if (delta === 0) return;
```

Comment:
fwiw, #1832 covers introducing ε for pan and rotation, but should probably be extended to zoom too.